ide/libs.git/src/org/netbeans/libs/git/GitClientCallback.java

```diff
@@ -54,7 +56,7 @@
  *     public Boolean askYesNoQuestion (String uri, String prompt);
  * };
  * client.{@link GitClient#setCallback(org.netbeans.libs.git.GitClientCallback) setCallback(myCallback)};
- * List<String> refspecs = Arrays.asList("refs/heads/*:refs/remotes/origin/*");
+ * List&lt;String&gt; refspecs = Arrays.asList("refs/heads/*:refs/remotes/origin/*");
  * client.fetch("http://myrepositoryhost/path", refspecs, pm);
  * </pre>
```

Comment:
you should be able to use:
```
<pre>{@code 

....new ArrayList<>()....

}</pre>
```
`@code` is there to escape everything, `pre` maintains the formatting.